tests/xtro-sharpie/Makefile

```
@@ -172,7 +172,7 @@ else
 classify-maccatalyst: ; @true
 endif
 
-classify: build $(XIOS_PCH) $(XWATCHOS_PCH) $(XTVOS_PCH) $(XMAC_PCH) $(XCAT_PCH)
+classify: build $(if $(INCLUDE_IOS), $(XIOS_PCH)) $(if $(INCLUDE_TVOS), $(XTVOS_PCH)) $(if $(INCLUDE_WATCH), $(XWATCHOS_PCH)) $(if $(INCLUDE_MAC), $(XMAC_PCH)) $(if $(INCLUDE_MACCATALYST), $(XCAT_PCH))
```

That's kind of unwieldy to read.

Maybe do something like this in each "ifdef INCLUDE_*" above:

```make
ifdef INCLUDE_IOS
PCH_FILES+=$(XIOS_PCH)
...
endif

ifdef INCLUDE_TVOS
PCH_FILES+=$(XTVOS_PCH)
...
endif

...
```

and then this becomes:

```suggestion
classify: build $(PCH_FILES)
```